examples/dialogflowcx-private-webhook-example/docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: "3.3"
```

Comment:
I am not sure if this file needs a license header or not (same with the Dockerfile actually). If this is "complex enough" and not trivial, I would add a license header.

Reply:
done